src/index.d.ts

```diff
@@ -374,6 +374,7 @@ interface ChannelListProps {
   renderUserProfile?: (props: RenderUserProfileProps) => React.ReactNode;
   renderHeader?: (props: void) => React.ReactNode;
   queries?: ChannelListQueries;
+  ableAutoSelectChannelItem?: boolean;
```

Comment:
We decide to rename this, right? Anyways, it is better to make this a separate PR(becuase its a feature), but no issue